pyquil/device/tests/test_graph.py

```diff
@@ -0,0 +1,41 @@
+import networkx as nx
+from pyquil.device.transformers.graph_to_compiler_isa import (
+    compiler_isa_to_graph,
+    DEFAULT_2Q_GATES,
+    DEFAULT_1Q_GATES,
+)
+from pyquil.device.transformers import graph_to_compiler_isa
+from pyquil.device.graph import NxDevice
+
+
+def test_isa_from_graph_order():
```

Comment:
Please add brief docstrings for any new tests. This makes the test suite a little more readable at a glance and can help make sure that a test continues to fulfill its original purpose across changes